File: Dinesh.2508/java/flowcharts/FCP017.java
package DomJudge;

import java.util.Scanner;

public class FCP017{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int A = scanner.nextInt();
        int B = scanner.nextInt();
        int C = scanner.nextInt();

        int secondMax;

        if ((A > B && A < C) || (A > C && A < B)) {
            secondMax = A;
        } else if ((B > A && B < C) || (B > C && B < A)) {
            secondMax = B;
        } else {
            secondMax = C;
        }

        System.out.println(secondMax);
    }
}

File: SreeGanesh.2531/Python/FlowchartCode/FCP017.py
A=int(input())
B=int(input())
C=int(input())
if(A>=B and A<=C)or (A<=B and A>=C):
    second_max=A
elif(B>=C and B<=A)or(B<=C and B>=A):
    second_max=B
else:
    second_max=C

print(second_max)        